applications/deployments/deployment-strategies.adoc

```diff
@@ -3,6 +3,7 @@
 include::modules/common-attributes.adoc[]
 :context: deployment-strategies
 
+
```

Comment:
Some of the assemblies now have two preceding blank lines. Is it worth only adding blank lines to the assemblies that do not current have one between the header and the `toc` declaration, for consistency?

Reply:
Yep, I've noticed this for the couple of serverless ones I've looked at so far also

Reply:
I am trying to script a solution for this.

Reply:
I ran the script against a local clone of this PR and then pushed the changes to the original commit. There is now one blank line preceding the `toc::[]` line in each file.

Reply:
Thank you @pneedle-rh